librz/core/cmd/cmd_analysis.c

```diff
@@ -6139,6 +6139,154 @@ RZ_IPI RzCmdStatus rz_analysis_function_until_handler(RzCore *core, int argc, co
 	return RZ_CMD_STATUS_OK;
 }
 
+static int var_comparator(const RzAnalysisVar *a, const RzAnalysisVar *b) {
+	return (a && b) ? (a->delta > b->delta) - (a->delta < b->delta) : 0;
+}
+
+static void core_analysis_var_list_show(RzAnalysis *analysis, RzAnalysisFunction *fcn, int kind, RzOutputMode mode, PJ *pj) {
```

Comment:
While you are on it, could you please refactor this function to use RzCmdStateOutput?